### PR TITLE
Op 141/sylius upgrade

### PR DESCRIPTION
**Pull request by gabriela-lubkowska**
- Add support for Sylius 1.12
- Update README

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -8,36 +8,31 @@ on:
...
+                php: [ "8.0" ]
+                symfony: ["^5.4", "^6.0"]
+                sylius: ["~1.11.0", "~1.12.0"]
+                node: ["^14.17.x"]
+                mysql: [ "5.7", "8.0" ]
 
                 exclude:
-                    - php: 8.0
-                      sylius: ~1.9
+                    - # Segmentation fault while clearing cache
```
Comment (senghe) [inline .github/workflows/build.yml:30]: Is still current? As I remember, @rafalmal removed that conflict in another PRs and it passed.

File: tests/Application/.env [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # https://symfony.com/doc/current/best_practices/configuration.html#infrastructure-related-configuration
 
 ###> symfony/framework-bundle ###
-APP_ENV=test
+APP_ENV=dev
```
Comment (senghe) [inline tests/Application/.env:6]: Is this used in build? If yes, shouldn't it be `test`?

Comment (gabriela-lubkowska) [reply tests/Application/.env]: Changed back to test

File: .github/workflows/build.yml (hunk 1 above)
Comment (gabriela-lubkowska) [reply .github/workflows/build.yml]: Removed in next commit

**Pull request merged**